.travis.yml

```diff
@@ -1,3 +1,12 @@
 language: ruby
 rvm: 2.3.1
-bundler_args: --without production
\ No newline at end of file
+bundler_args: --without production
+
+deploy:
+  provider: heroku
+  api_key:
+    secure: ENCRYPTED_HEROKU_API_KEY_HERE
+  app: disease-info-api
+  on:
+    repo: devcenter-square/disease-info
+    branch: develop
```

Comment:
Okay, then I need to refactor that out of the code since the default is definitely master